client/boot/common.js

```diff
@@ -422,10 +420,8 @@ function renderLayout( reduxStore, reactQueryClient ) {
 const boot = async ( currentUser, registerRoutes ) => {
 	saveOauthFlags();
 	utils();
-	await loadPersistedState();
```

Comment:
You probably have a good reason to do so, but why are you removing the loading/persisting of the state of react-query?